src/pages/unsubscribe/index.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Link, navigate } from 'gatsby'
+import { Button } from 'components/form'
+import { localize, WithIntl } from 'components/localization'
+import Layout from 'components/layout/layout'
+import CheckIcon from 'images/common/check_icon.png'
+import device from 'themes/device'
+import { useDerivApi } from 'components/hooks/use-deriv-api'
+import { queryParams } from 'common/utility'
+import { decode } from 'common/url-base64-functions'
+
+const UnsubscrubeWrapper = styled.div`
```

Comment:
Pls fix the typo` UnsubscrubeWrapper`

Reply:
Could you share what kind of errors you noticed @amina-deriv? Unfortunately locally I haven't any

Reply:
i mean the spelling `UnsubscrubeWrapper` . It should be `UnsubscribeWrapper`. right?

Reply:
Ohh my 😄 Should be, thanks

Reply:
I think it's not resolved yet right?